### ISSUE TITLE
which one of ASL functions in the task of detection should be used?

### ISSUE THREAD

**Issue by y78h11b09** [Alibaba-MIIL/ASL#70]
In the task of ATSS detection ,  should we select which one of ASL  functions, ASLsinglelabel or AsymmetricLoss？， there are some problems when I directly use the AsymmetricLoss.

**Comment by mrT23** [comment]
class AsymmetricLoss(nn.Module)

this is the multi-label variant.

detection is a multi-label problem, with extra localization data

**Comment by mrT23** [comment]
- start from the original focal loss, with exactly the same alpha as theirs (i thought it was 0.7, maybe i was wrong)
- reproduce their results
- introduce asymmetry

**Comment by y78h11b09** [comment]
Ok, thank you very much.  if I have  better result,  I will show you my fixed things and configs.



***@***.***
 
From: mrT23
Date: 2021-10-30 14:07
To: Alibaba-MIIL/ASL
CC: y78h11b09; Mention
Subject: Re: [Alibaba-MIIL/ASL] which one of ASL functions in the task of detection should be used? (#70)
start from the original focal loss, with exactly the same alpha as theirs (i thought it was 0.7, maybe i was wrong)
reproduce their results
introduce asymmetry
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
